.evergreen/scripts/abi-stability-setup.sh

```diff
@@ -115,9 +115,9 @@ echo "Building old libraries..."
     -B build/old \
     -DCMAKE_INSTALL_PREFIX=install/old \
     -DBUILD_VERSION="${old_ver:?}-base" \
-    "${configure_flags[@]:?}"
-  "${cmake_binary:?}" --build build/old
-  "${cmake_binary:?}" --install build/old
+    "${configure_flags[@]:?}" || exit
```

Comment:
I expect the updated blocks may need to be updated to `(` and `)` instead of `{` and `}`. The `exit` appears to exit the shell without running the `cat old.log 1>&2` command.

Reply:
Good catch. Applied as suggested.